joomla/joomla-cms
# Contact modified to be validable and accessible

### Summary of Changes
Added missing < label > to < select > element, for accessibility.
Replaced align attribute on < img > element with css styling (vertical-align : middle).
Added missing < dt > child element inside < dl > element

### Testing Instructions
Joomla! 3.7.4
Create 2 Contacts and populate Image and Position fields (in default category).
Create a menuitem of type "list Contacts in a category" (default category) and in "Contact Display options" set "Show Contact List" ("Contact Position" and "Image" are set to "show" already).
Browse to that page and select one of the contacts, so a contacts selector is shown.

### Expected result
Validating HTML (https://validator.w3.org/) and accessibility check (https://achecker.ca/checker/index.php) should succeed.

### Actual result
Accessibility error: select element missing an associated label.
HTML Validation errors: The align attribute on the img element is obsolete. Use CSS instead.  /   Element dl is missing a required child element.

### Documentation Changes Required
none

### Notes
I think this is not a bug, but this PR is a little help toward html conformance and accessibility.
